### DISCUSSION TITLE
Push from gh

### DISCUSSION THREAD

**Discussion by jgedri** [community/community#68133] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

I have a (hopefully) simple question: Can I push from a local repo to a remote with gh? If not, is it a planned feature?

**Comment by seyLu** [comment, marked as answer]
#### Hi @jgedri,

There are no gh commands to commit nor push. You can refer to [this earlier discussion](https://github.com/cli/cli/discussions/6889), where the same question was asked.

**Comment by ghost** [comment]
Hi team,
I am trying to push through gh auth login after everything last step is the git push but i am not able to push 
![Screenshot 2024-05-18 200023](https://github.com/community/community/assets/137860091/d574eecf-996f-4e7d-9e27-cf44dbf3d3fb)
